net-firewall/fwbuilder/fwbuilder-6.0.0_p20200502.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == *9999 ]]
+then
+	EGIT_REPO_URI="https://github.com/fwbuilder/fwbuilder"
+	inherit git-r3
+else
+	inherit vcs-snapshot
+	commit=a5e14a966447c63bcf7b52a0202149e76bd5ed4a
+	SRC_URI="https://github.com/fwbuilder/fwbuilder/archive/${commit}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+fi
```

Comment:
Please remove the conditionals for -9999 package, since there is none and won't be. Rest looks good.